src/main/java/gov/cabinetoffice/gapfindapiadmin/services/ApiKeyService.java

```diff
@@ -118,4 +118,20 @@ public List<String> getFundingOrgForAllApiKeys() {
         return apiKeyRepository.findByUniqueFundingOrganisationNames();
     }
 
+    public NavBarDto generateNavBarDto() {
```

Comment:
I think all of these methods perhaps should move into their own service class as they don't really have anything to do with API keys

Reply:
moved them in the controller. has was the only scope where needed